src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -190,6 +188,40 @@ public int getRollupCountForCriteriaGroups(
         .get(0);
   }
 
+  private SnapshotBuilderDomainOption getDomainOptionFromSettings(String domainId, UUID datasetId) {
```

Comment:
What would you think about adding a `byName` to the end of this method name? Technically we can also get domain option via the ID of the domain concept (which is an int)

Reply:
Alternatively, you could overload this method to also have a version that takes an int, and that checks against ID.

Reply:
Yeah, I can create another method that takes an int, that will be helpful once searchConcepts uses an int instead of a string